SocialCareCaseViewerApi/V1/Factories/ResponseFactory.cs

```diff
@@ -80,24 +80,16 @@ public static List<BsonDocument> HistoricalCaseNotesToDomain(List<CaseNote> case
 
         public static List<BsonDocument> HistoricalVisitsToDomain(List<Visit> visits)
         {
-            List<BsonDocument> historicalVisits = new List<BsonDocument>();
-
-            foreach (var visit in visits)
-            {
-                historicalVisits.Add(new BsonDocument(
-                        new List<BsonElement>
-                        {
-                                new BsonElement("_id", visit.Id ?? ""),
-                                new BsonElement("worker_email", visit.CreatedByEmail),
-                                new BsonElement("form_name_overall", "Historical_Visit"),
-                                new BsonElement("form_name", visit.Title),
-                                new BsonElement("timestamp", visit.CreatedOn.ToString("dd/MM/yyyy H:mm:ss")), //format used in imported data so have to match for now
-                                new BsonElement("is_historical", true) //flag for front end
-                        }
-                        ));
-            }
-
-            return historicalVisits;
+            return visits.Select(visit => new BsonDocument(new List<BsonElement>
+                {
+                    new BsonElement("_id", visit.VisitId),
+                    new BsonElement("worker_email", visit.CreatedByEmail),
+                    new BsonElement("form_name_overall", "Historical_Visit"),
+                    new BsonElement("form_name", "Historical Visit"),
+                    new BsonElement("timestamp", visit.ActualDateTime ?? visit.PlannedDateTime),
```

Comment:
Dates have been a bit of a headache because we are importing data as well and that can have different formats. They mainly use "dd/MM/yyyy H:mm:ss" format, so just need to check that search filters etc. still work OK. We have a ticket for normalising data across the board.

Reply:
The datetime coming from platform API is in the iso-8601 format as a string which is what we tend to aim for.